indexer/services/ender/src/validators/order-fill-validator.ts

```diff
@@ -83,7 +85,7 @@ export class OrderFillValidator extends Validator<OrderFillEventV1> {
           this.block,
           indexerTendermintEvent,
           txId,
-          orderFillEventWithLiquidity,
+          orderFillEventV1ToOrderFill(orderFillEventWithLiquidity),
```

Comment:
nit: The return type for `createHandlers` should be `Handler<OrderFillWithLiquidity>` and not `Handler<OrderFillEventV1>`.

Reply:
done